cloudant-client/src/test/java/com/cloudant/api/query/IndexLifecycleTest.java

```diff
@@ -49,14 +47,14 @@
  */
 public class IndexLifecycleTest {
```

Comment:
Something for a future PR to possibly add clarity in the logs: adding `@DisplayName` for test classes e.g. `@DisplayName("Index lifecycle test...")`

Reply:
Probably future PR, but it would be useful